src/test/coverage_test.go

```diff
@@ -12,10 +12,11 @@ import (
 var target = &core.BuildTarget{Label: core.BuildLabel{PackageName: "src/test", Name: "coverage_test"}}
 
 const (
-	pythonCoverageFile    = "src/test/test_data/python-coverage.xml"
-	goCoverageFile        = "src/test/test_data/go_coverage.txt"
-	goCoverageFile2       = "src/test/test_data/go_coverage_2.txt"
-	goCoverageFile3       = "src/test/test_data/go_coverage_3.txt"
+	pythonCoverageFile = "src/test/test_data/python-coverage.xml"
+	// TODO(peterebden): Remove the 'x' on these once we update go-rules again
```

Comment:
I think there's an issue here where the coverage variables we create have the prefix `goCover` and so do these; they later get somehow detected as coverage vars. I guess something in 1.23 must have changed to find them.

The easiest / more correct fix is probably to make go-rules use a less usual coverage variable name. I'm changing these for now so I don't have to block on another round of updates there.